config/dev.config

```diff
@@ -58,15 +58,6 @@
                ]}
              ]}
            ]},
-           {epoch_pow_cuckoo_lager_event, [
```

Comment:
@lucafavatella have you actually tried the logs are still there? My understanding was that this handler should stay and the traces are redirected into it based on the file `log/aeternity_pow_cuckoo.log` in the handler and the same must be set for `lager_file_backend` in the trace.

Reply:
Traces are essentially redirections of log output. However, the sink is defined as part of the trace definition and must not exist before.

Reply:
> have you actually tried the logs are still there?

@jur0 I did not check manually. This was based on code inspection.

I am giving priority to other PRs.

Reply:
I checkout the branch, made a prod-package, started it with mining and got the expected log file (and only the expected log file) including content.